src/index.js

```diff
@@ -18,6 +18,7 @@ import tr from 'react-intl/locale-data/tr'; // Turkish
 import uk from 'react-intl/locale-data/uk'; // Ukrainian
 import ru from 'react-intl/locale-data/ru'; // Russian
 import it from 'react-intl/locale-data/it'; // Italian
+import mr from 'react-intl/locale-data/mr'; // Italian
```

Comment:
Italian?

Reply:
It should be Marathi, Updating it, Sorry for that.